### PR TITLE
 - Add CentOS 7 instructions for basic install

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: install/basic/index.rst [hunk 1]
```diff
@@ -58,6 +60,43 @@ Docker Setup (First time only)
...
+
+.. code-block:: shell
+
+  sudo yum install -y yum-utils device-mapper-persistent-data lvm2
+  sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+  sudo yum install docker-ce docker-ce-cli containerd.io
+  sudo systemctl start docker
+
+  sudo usermod -aG docker <your_system_user>
+  su <your_system_user>
```
Comment (t-book) [inline install/basic/index.rst:101]: `sudo usermod -aG docker ${USER}`

(to streamline the command a bit?)

**Comment by t-book** [comment]
Nice @afabiani looks good to me. Only one little optional thing,  ;)
https://github.com/GeoNode/documentation/pull/38#pullrequestreview-421693645

**Comment by t-book** [comment]
... in case you think it needed that I retry double test the install in a centos instance drop me a line.

**Comment by afabiani** [comment]
@t-book thanks, I just tested on a new centos 7 machine

Not all the packages have been found, but the setup works anyway. Probably it is ok for the time being.

If you have resources to test and improve, you are very welcome.

**Pull request merged**